expression/schema_test.go

```diff
@@ -20,16 +20,14 @@ import (
 	"github.com/pingcap/tidb/model"
 )
 
-func (s *testEvalSuite) TestSchemaString(c *C) {
-	colCount := 5
-	keyInfoCount := 3
+func GenerateSchema(colCount int, dbName, tblName, fromID string) *Schema {
```

Comment:
We only use it in this file,
may be this function can be unexported.

Same as line43. @zz-jason